server/apps/immich/src/config/asset-upload.config.ts

```diff
@@ -29,7 +29,8 @@ export const assetUploadOption: MulterOptions = {
         return;
       }
 
-      const originalUploadFolder = `${basePath}/${req.user.id}/original/${req.body['deviceId']}`;
+      // const originalUploadFolder = `${basePath}/${req.user.id}/original/${req.body['deviceId']}`;
+      const originalUploadFolder = join(basePath, req.user.id, 'original');
```

Comment:
I would still like to keep the `deviceId` sub folder. In case I want to move the photos from one device to another location. Let's fix this and I will merge it in

Reply:
Ok. It not important for my usecase.